gst/nnstreamer/tensor_filter/tensor_filter.c

```diff
@@ -721,6 +785,7 @@ gst_tensor_filter_transform (GstBaseTransform * trans,
   GST_TF_FW_INVOKE_COMPAT (priv, ret, invoke_tensors, out_tensors);
   if (need_profiling)
     record_statistics (priv);
+  track_latency (self);
```

Comment:
Do we need to track this while we are not profiling anything?
This appears to be an unnecessary workload for most deployed systems.

Isn't it ok to have this inside ```if (need_profiling)```?

Reply:
Update done in last changeset.